packages/common/src/PublicNetworks.js

```diff
@@ -3,6 +3,7 @@
 // ID as the network (i.e. Rinkeby will return 4 as the chainId), but some networks with chainID's > 255 need to
 // override the default behavior because their network ID is too high.
 const BRIDGE_CHAIN_ID = {
+  1337: 253,
```

Comment:
Had to add this to make deployments work for localhost ganache networks.

Reply:
nice thanks!